config/vufind/EDS.ini

```diff
@@ -403,3 +403,14 @@ AU = None
 ; the LibGuides URLs if the same name is present here.
 ;url[Expensive Science Database] = https://some.url
 ;url[Cool Humanities Database] = https://another.url
+
+; Define whether to display 'Short' or 'Long' author data on the EDS search result list page.
+; - 'Short' is fetched from the RecordInfo section of a record (the 
+; 'BibRecord/BibRelationships/HasContributorRelationships/*PersonEntity/Name/NameFull' data).
+; - 'Long' is fetched from the item 'Au' group data.
+; If you do set 'ResultListPage' to 'Short' then you can define how many author names to 
+; display in AuthorNameLimit. If a record has more author names than you define in AuthorNameLimit
+; then VuFind will display the more_authors_abbrev text.
+;[AuthorResultListInfo]
```

Comment:
It might be worth uncommenting the section heading even if leaving the settings commented. It's a common mistake that people will uncomment settings but forget to uncomment sections, so having the section there to begin with can prevent minor annoyance. :-)

Reply:
thanks @demiankatz I've added that in my latest follow-up!